content/soarian/dstu2.md

```diff
@@ -0,0 +1,69 @@
+
+The initial interaction between the user, Application, and API is to establish a common patient identity and link that identity to an Application user.  The client must have a patient-facing identity provider (IdP) capable of the SAML interaction with the API services. One and only one IdP per client for all API instances is supported.  Application users will establish user name and password in the IdP.  The Soarian proxy and token management capabilities will be enabled and the Application users will enter their temporary linking token in the Application to establish an association between the user ID and the patient.  
+The Application interacts with the API using OAuth2 and the IdP with SAML for trusted user connections.  When the patient is seen for a healthcare encounter, Soarian Clinicals establishes a temporary linking token which is provided to the patient. Individual tokens may also be assigned to patient-authorized proxy(ies). The Soarian Clinicals user is responsible for state and federal guidelines for interacting with the patient and managing proxies as entered in Soarian. The Application must provide a means for the Application user to establish themselves in an approved identity server and then enter the temporary linking token into the Application. The Application interacts with an authorization server to establish a trusted connection and user authentication and then passes the temporary linking token to the Soarian Clinicals ActivateLinkToken API and Soarian Clinicals establishes a user-patient relationship that is used in subsequent interactions. This interaction provides the Application a trusted link between the user definition used for authentication and the identity of that user as a Soarian managed patient or proxy thus authorizing each user for a specific patient's data. Each subsequent interaction between the Application and API utilizes the trusted connection and a patient scope selection that Soarian has authorized via the token exchange.
+
+The below figure shows the interaction summary between Application, IdP, Authorization server and Soarian API. 
+
+![Soarian API and Application interaction](/images/soarian-api-application-interaction.jpg)
```

Comment:
@daliboz you are correct.  Amaresh will update the diagram and send to @slvsridhar to update the documentation.   @MaxPhilips, we found that by expanding the browser window size (ctrl +) the text became much easier to read.